source/documentation/getting_started/quick_start.md

```diff
@@ -1,10 +1,59 @@
 ## Quick start guide
 
-Registers in beta are reliable and are ready for use in products and services.
+Registers which are in beta are reliable and ready for use in products and services. Each register has an open, RESTful API which you can use to access the data in the register. 
 
-1. [Find the base URL for the register(s) you want to use](https://registers.cloudapps.digital/registers). For example, for the `local-authority-eng` register: `https://local-authority-eng.register.gov.uk/`.
-2. [Find the endpoints you want](#apireference). For example, `/record/BIR` in `https://local-authority-eng.register.gov.uk/record/BIR`.
-3. Add `.json`, `.yaml`, `.csv`, `.tsv`, or `.ttl` to the end of the request URL (before any query parameters) to choose the format of the response, or amend headers to `accept application/json` or equivalent.
-4. Parse register data and use it in your product or service.
+You can follow this guide to quickly get started with GOV.UK Registers APIs. 
+
+**1. Find the base URL for the register(s) you want to use.** 
+
+You can find a full list of register base URLs [here](https://registers.cloudapps.digital/registers).
```

Comment:
I'm probably picky here but, is the "here" link a good practice in terms of accessibility? Back in the day links had to aim to be meaningful without contextual information so in this case the link would've been "list of register base URLs"

Reply:
Aside from the content the linked page doesn't show "base URL's"

Reply:
@arnau Yes, this was just a case of me trying to cram in too much last night. You're right.  @danmitchell-'s correct as well. I'm inclined to think it would actually be clearer to get rid of this altogether and for me to write a new sentence which explains it better from context, without giving a list of base URLs.

Reply:
👍 to @danmitchell- suggestion